src/main/java/com/google/codeu/data/Message.java

```diff
@@ -25,20 +25,24 @@
   private String user;
   private String text;
   private long timestamp;
+  private String recipient; // Added recipient
 
   /**
```

Comment:
Edit this specific javadoc to include the parameter {@code recipient} rather than adding a comment via //